### DISCUSSION TITLE
Can there be support for multiple projects?

### DISCUSSION THREAD

**Discussion by heyarviind** [pocketbase/pocketbase#180] [Q&A]
I build many tiny projects, if I use Pocketbase, I will need 2 servers for each (I know I can deploy on the same server but I am more comfortable with CI/CD on render.com) one for the Pocketbase backend and the other for my frontend.

If there is support for multiple projects, I can boot a shared server for Pocketbase and deploy my frontend app on other servers.

**Comment by ganigeorgiev** [comment]
This was requested before, but at least for v1.0.0, I'm not planning adding support for multiple projects (aka. having isolated group of users, collections, records, rules, etc.).

It will complicate a lot of things, especially when used as a framework. Even the web api will require at least everything to be prefixed with the project key or sending some custom header. A lot of questions also will arise - should every project maintain its own db and migrations or everything should be kept in one big db file,  should there be a separate settings per project, will the auth providers be shared, what if someone wants to associate 2 collections from different projects, etc.

**Reply by taylorgibb** [reply]
Thanks for all your hard work, just to chime in, this is also something i would be interested in.

**Reply by eqoram** [reply]
the pockethost project (https://github.com/benallfree/pockethost) might be of interest for you then :)

**Comment by dawnpatrol04** [comment]
Requesting this.    every project has it's own every everything...  all totally separated.
